smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/ServiceBareBonesClientGenerator.java

```diff
@@ -310,42 +305,10 @@ private void generateClientDefaults() {
                     + "trait of an operation.");
             writer.write("disableHostPrefix?: boolean;\n");
 
-            // feat(experimentalIdentityAndAuth): write httpAuthSchemes and httpAuthSchemeProvider into ClientDefaults
-            if (settings.getExperimentalIdentityAndAuth()) {
-                writer.addDependency(TypeScriptDependency.EXPERIMENTAL_IDENTITY_AND_AUTH);
-                writer.addImport("HttpAuthScheme", null, TypeScriptDependency.EXPERIMENTAL_IDENTITY_AND_AUTH);
-                writer.writeDocs("""
-                    experimentalIdentityAndAuth: Configuration of HttpAuthSchemes for a client which provides \
-                    default identity providers and signers per auth scheme.
-                    @internal""");
-                writer.write("httpAuthSchemes?: HttpAuthScheme[];\n");
-
-                String httpAuthSchemeProviderName = service.toShapeId().getName() + "HttpAuthSchemeProvider";
-                writer.addImport(httpAuthSchemeProviderName, null, AuthUtils.AUTH_HTTP_PROVIDER_DEPENDENCY);
-                writer.writeDocs("""
-                    experimentalIdentityAndAuth: Configuration of an HttpAuthSchemeProvider for a client which \
-                    resolves which HttpAuthScheme to use.
-                    @internal""");
-                writer.write("httpAuthSchemeProvider?: $L;\n", httpAuthSchemeProviderName);
-            }
-
             // Write custom configuration dependencies.
             for (TypeScriptIntegration integration : integrations) {
                 integration.addConfigInterfaceFields(settings, model, symbolProvider, writer);
             }
-
-            // feat(experimentalIdentityAndAuth): write any HttpAuthScheme config fields into ClientDefaults
-            // WARNING: may be changed later in lieu of {@link TypeScriptIntegration#addConfigInterfaceFields()},
-            // but will depend after HttpAuthScheme integration implementations.
-            if (settings.getExperimentalIdentityAndAuth()) {
```

Comment:
This is moved into `AddHttpAuthSchemeMiddleware::customize()` where the following interfaces and function are generated:

- `HttpAuthSchemeInputConfig`
- `HttpAuthSchemeResolvedConfig`
- `resolveHttpAuthSchemeConfig`